src/parsers/graph/model-graph-algo.cpp

```diff
@@ -338,6 +348,28 @@ namespace pinocchio
           model.addBodyFrame(target_vertex.name, j_id, body_pose);
         }
 
+        void operator()(const JointSpline & joint, const BodyFrame & b_f)
+        {
+          if (!edge.forward)
+            PINOCCHIO_THROW_PRETTY(
+              std::invalid_argument, "Graph - JointSpline cannot be reversed.");
```

Comment:
We have to think about that.

Right now I'm not very confident about the reversal of this joint.
I don't think creating a ctrlFrames vector with inverse transformation inside will give us the inverse transformation.